mobly/controllers/android_device.py

```diff
@@ -995,16 +995,21 @@ def take_bug_report(self,
     self.log.debug('Bugreport taken at %s.', full_out_path)
     return full_out_path
 
-  def take_screenshot(self, destination):
+  def take_screenshot(self, destination, filename=None):
     """Takes a screenshot of the device.
 
     Args:
       destination: string, full path to the directory to save in.
+      filename: string, additional name of the screenshot will be added after the
+        file_type:screenshot if given.
```

Comment:
should not refer to `file_type` as it is not relevant to users using this method?